### PR TITLE
Improve wording of contribute quest docs

### PR DESCRIPTION
**Pull request by arrival-spring**
I've read through and adjusted anything that didn't sound right, as a native English speaker.

Mostly minor thing like missing articles or word order.

### CODE REVIEW

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 1]
```diff
@@ -64,21 +64,21 @@ Use <kbd>Ctrl</kbd>+<kbd>N</kbd> to go to a certain class. <kbd>Ctrl</kbd>+ mous
...
 
-Are you trying to implement a quest that will have simple yes/no answer? Take [`AddBusStopLit`](app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_lit/AddBusStopLit.kt) quest as a base. Or [`AddTracktype`](app/src/main/java/de/westnordost/streetcomplete/quests/tracktype/AddTracktype.kt) where the mapper will be selecting one of presented images.
+Are you trying to implement a quest that will have a simple yes/no answer? Take the [`AddBusStopLit`](app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_lit/AddBusStopLit.kt) quest as a base. Or [`AddTracktype`](app/src/main/java/de/westnordost/streetcomplete/quests/tracktype/AddTracktype.kt) where the mapper will be selecting from one of some presented images.
 
-Is it going to be asked for POIs and should be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
+Is it going to be asked for POIs and it should be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```
Comment (riQQ) [inline CONTRIBUTING_A_NEW_QUEST.md:77]:
```suggestion
Is it going to be asked for POIs and should it be disabled by default? [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

Comment (arrival-spring) [reply CONTRIBUTING_A_NEW_QUEST.md:77]: Actually I don't like the "it" I added now, I think I should revert that line change.

Comment (mnalis) [reply CONTRIBUTING_A_NEW_QUEST.md:77]: > Actually I don't like the "it" I added now, I think I should revert that line change.

Agreed, although as a non-native speaker I always find such multiple-rules (_"is it XXX and should YYYY be ZZZZ?"_)  in question form (instead of if/then form) somewhat hard to parse. 

I.e. something like:
```suggestion
If it is going to be asked for POIs and should be disabled by default, then [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

or even better (if the emphasis is supposed to be on disabled-by-default part):

```suggestion
If the quest which is going to be asked for POIs should be disabled by default, then [`AddWheelchairAccessBusiness`](app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt) may be a good base.
```

would be more readable to me.

**Comment by westnordost** [comment]
LGTM. Tell me when I shall merge this

File: CONTRIBUTING_A_NEW_QUEST.md (hunk 1 above)
Comment (arrival-spring) [reply CONTRIBUTING_A_NEW_QUEST.md]: First one sounded better to me, so I've gone with that

**Comment by arrival-spring** [comment]
Ready from my point of view

**Comment by matkoniecz** [comment]
thanks for improving my English!
